CHANGES.md

```diff
@@ -23,12 +23,16 @@ To be released.
 
 ### Bug fixes
 
+- `NetMQTransport.CreatePermission()` became to have timeout when locked for 2.5 seconds. [[#611], [#869]]
```

Comment:
I guess we don't need to mention about `NetMQTransport.CreatePermission()` because it hadn't opened to Libplanet users. 😕